chroma-manager.conf.template

```diff
@@ -292,7 +292,8 @@ server {
         proxy_set_header X-Forwarded-Host $host;
         proxy_set_header X-Forwarded-Server $host;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
-        proxy_pass {{HTTP_AGENT2_PROXY_PASS}}/message;
+        # /agent2/message proxy_pass
+        proxy_pass {{HTTP_AGENT2_PROXY_PASS}}/;
```

Comment:
Have you tested this outside docker? Does this still work?

Reply:
I am running a full test both in vagrant and docker and will post the results.